packages/block-library/src/button/color-edit.js

```diff
@@ -50,7 +52,7 @@ const cleanEmptyObject = ( object ) => {
 		: cleanedNestedObjects;
 };
 
-function ColorPanel( { settings, clientId, enableContrastChecking = true } ) {
+function ColorPanel( { settings, clientId, enableContrastChecking = isWebPlatform } ) {
```

Comment:
Hmm, I'm wondering if we could change it a bit to block the contrast checking for mobile even if it is set to `true` in props?

Reply:
We can add check `isWebPlatform` before each `enableContrastCheking`.